File: springboot/ps5GameTradingSystem/src/main/java/com/zxy/ps5gametradingsystem/entity/Recycle.java
package com.zxy.ps5gametradingsystem.entity;


import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("`recycle`")
public class Recycle {
    private  String id;
    private  String userId;
    private  String gameName;
    private  Integer valuation;
    private  String recyclingStatus;
    private  String version;
    private  String language;
    private  String appearance;
    private  String box;
}

File: vue/ps5GameTradingSystem/src/types/recycle.ts
export interface recycle{
    id:   string,
    userId: string,
    gameName: string,
    valuation: number,
    recyclingStatus: string,
    version: string,
    language: string,
    appearance: string,
    box: string,
}
export interface recycleItem {
    id:   string,
    userId: string,
    gameName: string,
    valuation: number,
    recyclingStatus: string,
    version: string,
    language: string,
    appearance: string,
    box: string,
    checked: boolean  // 前端专属的选中状态
}
export type recycles =recycle[]
export type recycleItems = recycleItem[];